tests/db_setup.py

```diff
@@ -64,6 +40,8 @@ def teardown(self):
         """
         self.session.query(LayerData).delete()
         self.session.query(SiteObservers).delete()
+        self.session.query(PointData).delete()
+        self.session.query(CampaignObservation).delete()
```

Comment:
These might need to change order here for hanging tests when multiple test classes run